pkg/clusterconfig/clusterconfig.go

```diff
@@ -91,7 +91,11 @@ func GetAWSConfig(kubeconfig *rest.Config, listers *regopclient.Listers) (*Confi
 		return nil, err
 	}
 
-	if infra.Status.PlatformStatus.Type == configapiv1.AWSPlatformType {
+	platformType := infra.Status.Platform
+	if infra.Status.PlatformStatus != nil {
+		platformType = infra.Status.PlatformStatus.Type
+	}
+	if platformType == configapiv1.AWSPlatformType {
```

Comment:
If infra.Status.PlatformStatus is nil, won't line 99 fail with a nil pointer?
`cfg.Storage.S3.Region = infra.Status.PlatformStatus.AWS.Region`

Reply:
ouch

Reply:
fixing

Reply:
i think you're correct, plus the other platformstatus usages you idenfied in slack will also be a problem

Reply:
fixed by reintroducing the install config inspection